### PR TITLE
Class Panel update

### PR DESCRIPTION
**Pull request by khaidoan2729**
Updated SQL query to get information for class panels. Managed to get all information in one entry and separate by commas. Still need to integrate user context for 'Status' and 'Volunteers' on the panel. Maybe discuss what to display if there is no information for certain fields for example scheduled time. The panel can be reused in the Schedule page but might need to wrap the code with the component (See frontend/src/pages/classes/index.js).

### CODE REVIEW

File: frontend/src/assets/fonts/Montserrat/Montserrat-Regular.woff2
Comment (jjessieshang) [inline frontend/src/assets/fonts/Montserrat/Montserrat-Regular.woff2:1]: Let's remove imported fonts, and just add their imports to the global styles.css file

File: frontend/src/assets/images/bc_brain_logo.png
Comment (jjessieshang) [inline frontend/src/assets/images/bc_brain_logo.png:1]: assets only contains images, so can remove the images folder

File: frontend/src/components/classPanel/index.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
+import "./index.css";
+import React, {useEffect, useState} from 'react';
+import button_icon_close from "../../assets/images/button-icons/button-icon-close.png";
+import button_icon_prev from "../../assets/images/button-icons/button-icon-prev.png";
+import button_icon_next from "../../assets/images/button-icons/button-icon-next.png";
+import { getClassById } from '../../api/classesPageService';
+
+
```
Comment (jjessieshang) [inline frontend/src/components/classPanel/index.js:8]: remove spaces

**Review by jjessieshang — COMMENTED** [review]
Functionality looks good - please work with Josh and Leo to integrate with the class page

**Review by leowang801 — COMMENTED** [review]
Lets fix the merge conflicts

**Pull request merged**